website/content/gateway/api_reference/changes/changes.md

```diff
@@ -10,6 +10,12 @@ Follow coming changes on the [source code repository](https://github.com/clearha
 
 Sorted by descending timestamp
 
+#### Croatia Kuna (HRK) changes to Euro (EUR)
```

Comment:
```suggestion
#### Croatian Kuna (HRK) changes to Euro (EUR)
```

Reply:
Stole the text from elsewhere without even reading it 🙈

Thanks for noticing! 🙇‍♂️